torchvision/prototype/features/_feature.py

```diff
@@ -137,6 +140,28 @@ def _F(self) -> ModuleType:
             _Feature.__F = functional
         return _Feature.__F
 
+    # Add properties for common attributes like shape, dtype, device, ndim etc
+    # this way we return the result without passing into __torch_function__
+    @property
+    def shape(self):
+        return self.as_subclass(torch.Tensor).shape
```

Comment:
Have we tried maintaining the original reference of the input tensor `data` prior getting wrapped in the subclassed tensor? @ezyang proposed this idea offline as a means to speed things up. He just cautioned to keep this private because if a user gets hold of the original reference and do `original.resize_()` this change will be available only on that tensor and not on the wrapped one.

Reply:
Is this really faster than going into torch function? You have to do a tensor allocation here!